consensus/polybft/consensus_runtime.go

```diff
@@ -276,6 +276,11 @@ func (c *consensusRuntime) FSM() (*fsm, error) {
 		return nil, err
 	}
 
+	eventRoot, err := c.getExitEventRootHash(epoch.Number)
```

Comment:
Is it possible to integrate event root calculation to BuildProposal?

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/838/commits/a6cbda397c6a46283dc10fab61f673a0e1de169c